rpm/ndb/ndb.go

```diff
@@ -14,8 +14,9 @@ var le = binary.LittleEndian
 
 // Used throughout the various DBs.
 const (
-	slotSize  = 4 * 4
-	slotStart = 2
+	slotSize           = 4 * 4
+	slotStart          = 2
+	slotEntriesPerPage = 4096 / 16
```

Comment:
```suggestion
	slotEntriesPerPage = pageSz / slotSize
```

Please forgive my inconsistent naming.

Reply:
Although this is unused, so just remove it

Reply:
:facepalm: